spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestWriteMetricsConfig.java

```diff
@@ -271,28 +271,30 @@ public void testCustomMetricCollectionForNestedParquet() throws IOException {
       DataFile file = task.file();
 
       Map<Integer, Long> nullValueCounts = file.nullValueCounts();
-      Assert.assertEquals(3, nullValueCounts.size());
-      Assert.assertTrue(nullValueCounts.containsKey(longCol.fieldId()));
-      Assert.assertTrue(nullValueCounts.containsKey(recordId.fieldId()));
-      Assert.assertTrue(nullValueCounts.containsKey(recordData.fieldId()));
+      assertThat(nullValueCounts).hasSize(3);
+      assertThat(nullValueCounts).containsKey(longCol.fieldId());
+      assertThat(nullValueCounts).containsKey(recordId.fieldId());
+      assertThat(nullValueCounts).containsKey(recordData.fieldId());
 
       Map<Integer, Long> valueCounts = file.valueCounts();
-      Assert.assertEquals(3, valueCounts.size());
-      Assert.assertTrue(valueCounts.containsKey(longCol.fieldId()));
-      Assert.assertTrue(valueCounts.containsKey(recordId.fieldId()));
-      Assert.assertTrue(valueCounts.containsKey(recordData.fieldId()));
+      assertThat(valueCounts).hasSize(3);
```

Comment:
these call all be combined to a more fluent call: `assertThat(..).hasSize(3).containsKey(..).containsKey(..).containsKey(...)`

Reply:
updated